crates/bevy_gltf/src/loader.rs

```diff
@@ -282,6 +286,23 @@ async fn load_gltf<'a, 'b>(
                 mesh.set_indices(Some(Indices::U32(indices.into_u32().collect())));
             };
 
+            let target_count = reader.read_morph_targets().len();
+            if target_count != 0 {
+                let vertex_count = mesh.count_vertices();
+                if let Ok(MorphTargetImage { image, .. }) = MorphTargetImage::new(
+                    reader.read_morph_targets(),
+                    vertex_count as u32,
+                    target_count,
```

Comment:
I guess we'd need to check target_count and vertex_count against some wgpu Limits for texture dimensions.

Reply:
I do the check in the `MorphTargetImage::displacements_buffer` method, I return a `TooManyAttributes` error with a nice error message. in `crates/bevy_render/src/mesh/morph/mod.rs` I set the `MAX_TEXTURE_WIDTH` as 2048.